Keas.Core/Domain/Equipment.cs

```diff
@@ -22,7 +22,10 @@ public Equipment()
         public string ProtectionLevel { get; set; }
 
         [StringLength(2)]
-        public string AvailabilityLevel { get; set; }
+        public string AvailabilityLevel { get; set; }
+
+        [StringLength(16)]
+        public string BigfixId { get; set; }
```

Comment:
You had a good name earlier i don't remember, something like ExternalIdentifier or whatnot.  I don't think we should name the variable itself bigfix but just it's an id to whatever central system comes and goes